wordpress-mobile/WordPress-iOS
# Migrates BasePageListCell and RestorePageTableViewCell to Swift

This is part of a larger PR that was separated to keep the PRs simpler.

Migrates `BasePageListCell` and `RestorePageTableViewCell` preparing them for some changes to unify how post and page states are managed for better offline support.

## To test:

1. Check that the pages list renders fine.
2. Delete a page, restore it.
3. Play with the page states.

Known issues: the page list doesn't always refresh right, but this issue wasn't introduced by these changes.

## PR submission checklist:
